localization/ar_tag_based_localizer/README.md

```diff
@@ -18,10 +18,11 @@ This package includes two nodes.
 
 #### Input
 
-| Name                  | Type                           | Description  |
-| :-------------------- | :----------------------------- | :----------- |
-| `~/input/image`       | `sensor_msgs::msg::Image`      | Camera Image |
-| `~/input/camera_info` | `sensor_msgs::msg::CameraInfo` | Camera Info  |
+| Name                  | Type                                            | Description                     |
+| :-------------------- | :---------------------------------------------- | :------------------------------ |
+| `~/input/image`       | `sensor_msgs::msg::Image`                       | Camera Image                    |
+| `~/input/camera_info` | `sensor_msgs::msg::CameraInfo`                  | Camera Info                     |
+| `~/input/ekf_pose`    | `geometry_msgs::msg::PoseWithCovarianceStamped` | EKF Pose without IMU correction |
```

Comment:
Could you describe here or elsewhere that how the `ekf_pose` is used in this node?

Reply:
thank you :+1: